cartoframes/data/dataset/registry/dataframe_dataset.py

```diff
@@ -164,9 +164,9 @@ def _rows(df, dataframe_columns_info, with_lnglat):
             row_data.append(encode_row(val))
 
         if with_lnglat:
-            lng_val = row[with_lnglat[0]]
-            lat_val = row[with_lnglat[1]]
-            if lng_val and lat_val:
+            lng_val = df[with_lnglat[0]][index]
```

Comment:
`df.at` may be a more appropriate access method for this type of use case: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.at.html

Reply:
Cool thanks